### DISCUSSION TITLE
更新日志 - 22.03.7 - 20241025

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1869] [Announcements]
本次更新只涉及 x86 和部分 RK35xx 机型

# 设备
* RK35xx：新增支持友善电子 R3S，新增支持Radxa E52C #1858 #1820
* x86：集成两家万兆级网卡驱动，支持 Marvell(aQuantia) AQtion 系列，以及 QLogic QED 25/40/100Gb 系列。（驱动未使用情况下会自动从内存中卸载） #1860

**Comment by zzzbaba** [comment]
我以为2个多月憋了个大更

**Reply by jjm2473** [reply]
等上游发新版本呢

**Reply by deepinsc2020** [reply]
> 等上游发新版本呢

N1的坑啥时候填啊

**Reply by jjm2473** [reply]
@deepinsc2020 N1之前发的版本有啥问题吗？

**Reply by deepinsc2020** [reply]
> @deepinsc2020 N1之前发的版本有啥问题吗？

我看b站说要更新N1  说填坑

**Reply by jjm2473** [reply]
> 我看b站说要更新N1 说填坑

可能只是加上在线升级，实际上N1的代码没啥变化。

**Reply by hippieZhou** [reply]
同一个U盘，我刷 F大的 openwrt 一次就能U盘启动刷成功，但是isstoreos 怎么刷都不行，插上U盘，始终无法正常启动，不知道啥原因，不确定是我N盒子的问题还是镜像的问题

**Reply by newsharpstar** [reply]
> 同一个U盘，我刷 F大的 openwrt 一次就能U盘启动刷成功，但是isstoreos 怎么刷都不行，插上U盘，始终无法正常启动，不知道啥原因，不确定是我N盒子的问题还是镜像的问题

我现在不折腾了。直接上istore，在线更新，很方便。

**Comment by leonda2008** [comment]
我X86没有提示更新呢

**Comment by Linzi-GH** [comment]
![image](https://github.com/user-attachments/assets/6533555b-b801-483c-9f62-f8ab5f121c99)
更新失败

**Reply by jjm2473** [reply]
现在重试下，之前的固件被CDN缓存了不完整的文件，现在已经清理了。

**Reply by Linzi-GH** [reply]
可以了

**Comment by binfoo** [comment]
下个内核直接6.8吗

**Reply by jjm2473** [reply]
看openwrt的消息应该6.6吧

**Reply by breeze303** [reply]
Linux kernel都取消了6.8咋可能是6.8

**Comment by binfoo** [comment]
是的，记错了，以为和pve一个版本，op主线快照是6.6

**Comment by dson881103** [comment]
i226可以识别2.5g了吗

**Reply by daoyelihuowang** [reply]
一直可以啊

**Reply by jjm2473** [reply]
如果有问题，换网线试试，一般cat 5e的扁线就够了

**Comment by cxcxcx352364** [comment]
PVE直通AX210网卡有问题，没办法开5G频率的WiFi，有办法解决这个问题吗

**Reply by jjm2473** [reply]
Intel的wifi全部不支持5G AP，不用纠结了，上游都不支持。

**Comment by drama8888** [comment]
为什么X86的软件商店比arm版本的要少很多啊，比如RustDesk在X86商店就没有。

**Reply by jjm2473** [reply]
arm版本也没有啊。istore的维护页面里面可以显示所有架构的插件。

**Comment by breeze303** [comment]
24.10分支已经出了

**Comment by syj119** [comment]
这个版本和之前的版本都不支持i350网卡啊

**Comment by gzxiexl** [comment]
能否提供rootfs格式的固件 谢谢

**Comment by newsharpstar** [comment]
iStore软件商店打不开。
![image](https://github.com/user-attachments/assets/9173c976-9931-4d73-9cf6-f0f8a5177611)

点击更新软件列表报错。
![image](https://github.com/user-attachments/assets/7c179a5c-7656-462c-9e25-e3ae13352555)

**Reply by liqun5417** [reply]
一模一样的问题

**Reply by fejeIaqbhbv** [reply]
![i](https://github.com/user-attachments/assets/d78426d0-50a3-4c64-9bf7-042d3e7ca6f1)

**Comment by eunrui** [comment]
网口状态突然空白了是因为什么呢，网口设置里也是什么也看不到，`/cgi-bin/luci/istore/network/interface/config/`接口报错
![图片20241202175931](https://github.com/user-attachments/assets/7641f026-9710-42df-8408-49fd9bb57891)
![图片20241202175924](https://github.com/user-attachments/assets/81b27e16-a882-42a6-8a1d-134d33c482ed)
![图片20241202180114](https://github.com/user-attachments/assets/45326185-779f-4f5c-a640-7c22b7150e71)

**Reply by eunrui** [reply]
wol唤醒也不能用了，难受

**Reply by changandeyue** [reply]
你好我也遇到相同的问题，不过我的wol还是可以使用，但你描述的网络接口问题依旧存在，请问现在解决了吗？

**Reply by jjm2473** [reply]
执行这个命令看看`ip -pretty -detail -json link show`

**Reply by jjm2473** [reply]
这边有记录排查过程和解决方案 https://github.com/istoreos/istoreos/issues/1986

**Reply by eunrui** [reply]
> 你好我也遇到相同的问题，不过我的wol还是可以使用，但你描述的网络接口问题依旧存在，请问现在解决了吗？

是的修复了，https://github.com/istoreos/istoreos/issues/1986#issuecomment-2548204899

**Reply by eunrui** [reply]
> 这边有记录排查过程和解决方案 #1986

感谢!
